src/web/nootils.js

```diff
@@ -73,6 +73,20 @@ module.exports = function(settings) {
     isDateValid: function(date) {
       return !isNaN(date.getTime());
     },
+    getField: function(report, field) {
+      /* field is a string e.g 'dob' (equivalent to report.fields.dob) or
+      * 'screening.test_result' equivalent to report.fields.screening.test_result
+      */
+      return _.propertyOf(report.fields)(field.split('.'));
+    },
+    fieldEquals: function(report, field, value) {
```

Comment:
is this function helpful? seems about as easy to just write `Utils.getField(report, field) === value` vs `Utils.getField(report, field, value)`